src/main/java/com/minecolonies/coremod/client/gui/GuiMinecoloniesChest.java

```diff
@@ -0,0 +1,59 @@
+package com.minecolonies.coremod.client.gui;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.inventory.GuiContainer;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.inventory.ContainerChest;
+import net.minecraft.inventory.IInventory;
+import net.minecraft.util.ResourceLocation;
+
+/**
+ * ------------ Class not Documented ------------
+ */
+public class GuiMinecoloniesChest extends GuiContainer
+{
+    /**
+     * The ResourceLocation containing the chest GUI texture.
+     */
+    private static final ResourceLocation CHEST_GUI_TEXTURE = new ResourceLocation("minecraft:textures/gui/container/generic_54.png");
+    private final IInventory upperChestInventory;
+    private final IInventory lowerChestInventory;
+    /**
+     * window height is calculated with these values; the more rows, the heigher
+     */
+    private final int        inventoryRows;
+
+    public GuiMinecoloniesChest(IInventory upperInv, IInventory lowerInv)
+    {
+        super(new ContainerChest(upperInv, lowerInv, Minecraft.getMinecraft().player));
+        this.upperChestInventory = upperInv;
+        this.lowerChestInventory = lowerInv;
+        this.allowUserInput = false;
+        int i = 222;
+        int j = 114;
+        this.inventoryRows = lowerInv.getSizeInventory() / 9;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 9 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS109)